packages/clickhouse/pkg/events/delivery.go

```diff
@@ -0,0 +1,161 @@
+package events
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"go.uber.org/zap"
+
+	"github.com/e2b-dev/infra/packages/clickhouse/pkg/batcher"
+	"github.com/e2b-dev/infra/packages/shared/pkg/events"
+	flags "github.com/e2b-dev/infra/packages/shared/pkg/feature-flags"
+)
+
+const InsertSandboxEventQuery = `INSERT INTO sandbox_events
+(
+    timestamp,
+    sandbox_id, 
+    sandbox_execution_id,
+    sandbox_template_id,
+    sandbox_build_id,
+    sandbox_team_id,
+    event_data,
+    type,
+    version,
+    id
+)
+VALUES (
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?
+)`
+
+type ClickhouseDelivery struct {
+	batcher *batcher.Batcher[SandboxEvent]
+	conn    driver.Conn
+}
+
+func NewDefaultClickhouseSandboxEventsDelivery(ctx context.Context, conn driver.Conn, featureFlags *flags.Client) (*ClickhouseDelivery, error) {
+	maxBatchSize := 100
+	if val, err := featureFlags.IntFlag(ctx, flags.ClickhouseBatcherMaxBatchSize); err == nil {
+		maxBatchSize = val
+	}
```

Comment:
Can we rely on the feature flag default value, defined in shared/pkg/feature-flags/flags.go? If this feels strange, I'd be super happy with a small refactor to avoid returning an error in the case that the default value is used.


```suggestion
	maxBatchSize, _ := featureFlags.IntFlag(ctx, flags.ClickhouseBatcherMaxBatchSize)
```